frontend/packages/integration-tests-cypress/tests/crud/namespace-crud.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { testName } from '../../utils/common';
+import { listPage } from '../../views/list-page';
+import { modal } from '../../views/modal';
+
+before(() => {
+  cy.login();
+  cy.createProject(testName);
+});
+after(() => {
+  cy.deleteProject(testName);
+  cy.logout();
+});
+
+describe('Namespace', () => {
```

Comment:
This seems like a big step back from how the CRUD tests work today. We should have a single test scenario that loops over all the types and runs common actions on those types instead of having to code every resource separately. Otherwise we can't scale.

Reply:
yes, that is the plan, but Namespace CRUD was already broken out in the existing CRUD test, and we wanted a simple example to start with.

Reply:
> yes, that is the plan, but Namespace CRUD is already broken out in the existing CRUD test, and we wanted a simple example to start with.

I'd focus the Namespace test to the thing that's unique, the namespace dialog. Things like labels should be a general test and covered when we migrate crud or as a separate labels scenario.

Reply:
The labels are part of the 'U'pdate part of the Namespace CRUD test.